backend/app/models/project.py

```
@@ -49,6 +49,9 @@ class Project(ProjectBase, table=True):
     collections: list["Collection"] = Relationship(
         back_populates="project", cascade_delete=True
     )
+    openai_conversations: list["OpenAIConversation"] = Relationship(
```

Ruff (0.12.2)

Undefined name `OpenAIConversation`

(F821)